MAINTENANCE.md

```diff
@@ -58,6 +58,8 @@ It is also scheduled to run daily on the *master* branch.
 
 This is ran on every commit to the *master* and Pull Request branches.
 It is also scheduled to run daily on the *master* branch.
+In draft Pull Requests, only one job (Ubuntu + Python latest)
+is triggered to save on Continuous Integration resources.
 
```

Comment:
```suggestion
Alternatively, add [skip ci] to your commit message so that Github Actions won't run.
See https://github.blog/changelog/2021-02-08-github-actions-skip-pull-request-and-push-workflows-with-skip-ci/.

```

Reply:
Prefer to leave it to another PR, as more tests need to be done before documenting it.

Reply:
Ok, feel free to copy this text to that next PR.